src/axom/sidre/core/Group.cpp

```diff
@@ -1991,22 +2102,26 @@ Group* Group::createGroupAndLoad(std::string& group_name,
  *
  *************************************************************************
  */
-void Group::loadExternalData(const std::string& path)
+bool Group::loadExternalData(const std::string& path)
 {
   Node n;
   createExternalLayout(n);
+  ConduitErrorSuppressor checkConduitCall(getDataStore());
 
 #ifdef AXOM_USE_HDF5
   // CYRUS'-NOTE, not sure ":" will work with multiple trees per
   // output file
```

Comment:
Outside the scope of this PR, but is this still relevant/unresolved?

Reply:
Very good question.  Created #1254 to track.